jumanji/environments/routing/cmst/viewer.py

```diff
@@ -0,0 +1,352 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+from typing import Callable, Dict, List, Optional, Sequence, Tuple
+
+import chex
+import jax.numpy as jnp
+import matplotlib.animation
+import matplotlib.pyplot as plt
+import numpy as np
+from numpy.typing import NDArray
+
+import jumanji.environments
+from jumanji.environments.routing.cmst.types import State
+from jumanji.viewer import Viewer
+
+grey = (100 / 255, 100 / 255, 100 / 255)
+white = (255 / 255, 255 / 255, 255 / 255)
+yellow = (200 / 255, 200 / 255, 0 / 255)
+red = (200 / 255, 0 / 255, 0 / 255)
+black = (0 / 255, 0 / 255, 0 / 255)
+blue = (50 / 255, 50 / 255, 160 / 255)
+
+
+class Renderer(Viewer):
+    """Viewer class for the Cooperative Minimum Spanning Tree environment."""
+
+    def __init__(
+        self,
+        num_agents: int,
+        name: str = "CoopMinSpanTree",
+    ) -> None:
+        """Create a ConnectorRenderer instance for rendering the Connector environment.
+
+        Args:
+            num_agents: Number of agents in the environment.
+        """
+
+        self._name = name
+        self.num_agents = num_agents
+
+        # Pick display method. Only one mode avaliable
+        self._display: Callable[[plt.Figure], Optional[NDArray]]
+        self._display = self._display_human
+
+        np.random.seed(0)
+        random.seed(0)
```

Comment:
I don't think the `random` package from the standard library is needed here.

Reply:
This has been fixed